src/zope/traversing/tests/test_vh.py

```diff
@@ -31,9 +31,9 @@ def setTraversalStack(self, stack):
         self._traversal_stack[:] = list(stack)
 
     def setApplicationServer(self, host, proto='http', port=None):
-        host = "%s://%s" % (proto, host)
+        host = "{}://{}".format(proto, host)
         if port:
-            host = "%s:%s" % (host, port)
+            host = "{}:{}".format(host, port)
```

Comment:
(I would use f-strings for these.)

Reply:
This was changed by `pyupgrade`. I updated it to an f-string in https://github.com/zopefoundation/zope.traversing/pull/19/commits/92c8287b5a247fa30741ba71d05a2ded4c1261d6.